test/turbopack-dev-tests-manifest.json

```diff
@@ -8724,7 +8724,10 @@
       "use-cache-metadata-route-handler should generate sitemaps with a metadata route handler that uses \"use cache\""
     ],
     "failed": [],
-    "pending": [],
+    "pending": [
+      "use-cache-metadata-route-handler should include the client reference manifest in the route.js.nft.json files of dynamic metadata routes",
+      "use-cache-metadata-route-handler should not include the client reference manifest in the route.js.nft.json files of static metadata routes"
+    ],
```

Comment:
Why do these need to be added here as pending if they are only ever run for next start?